packages/ui/tests/screenshots/pages/accounts/accounts.base.screen.ts

```diff
@@ -113,3 +202,32 @@ test('Accounts - ACL', async ({page}) => {
     await page.waitForSelector('.g-dialog');
     await expect(page).toHaveScreenshot();
 });
+
+test('Accounts - List', async ({page}) => {
+    await page.goto(makeClusterUrl(`accounts/general?sortState=asc-false,field-name`));
+    await page.waitForSelector('.elements-table');
+    await page.fill('span[data-qa="accounts-name-filter"] input', 'e2e');
+
+    await accounts(page).prepareListPage();
+    await expect(page).toHaveScreenshot();
+
+    await accounts(page).changeListMode('select-popup-g-:rl:-item-1');
```

Comment:
Isn't it better to use human readable text here? Like:
- Default
- Disk space
- Nodes
- ...